scm/src/gmtb_scm_utils.F90

```diff
@@ -102,7 +118,7 @@ module NetCDF_read
   
   implicit none
   
-  real(kind=dp) :: missing_value = -9999.0
+  real(kind=sp) :: missing_value = -9999.0
```

Comment:
This is to match the variable type in the NetCDF files, correct? Means all data is written as float and not as double. Can you think of any variable that may need double precision accuracy in the output?

Reply:
Yes. Not at the moment, but it is easy enough to change if/when dp is required.